src_py/nerlPlanner/JsonElementWorkerDefinitions.py

```diff
@@ -81,7 +81,9 @@
     ("Probabilistic" , ProbabilisticActivationFunctionMap),
     ("LSTM" , None),
     ("Reccurrent" , None),
-    ("Unscaling" , UnScalingMethodMap)]
+    ("Unscaling" , UnScalingMethodMap),
+    ("Flatten" , FlattenMethodMap),
```

Comment:
@Orisadek 
You should validate that Flatten and Bounding are supported by parser and NerlWorkerNN